content/en/docs/tasks/administer-cluster/migrating-from-dockershim/migrate-dockershim-dockerd.md

```diff
@@ -76,6 +76,8 @@ instructions for that tool.
 1.  Open `/var/lib/kubelet/kubeadm-flags.env` on each affected node.
 1.  Modify the `--container-runtime-endpoint` flag to
     `unix:///var/run/cri-dockerd.sock`.
+1.  Modify the `--container-runtime` flag to `remote`
+    (available in Kubernetes 1.26 and earlier).
```

Comment:
It is not just "available", right?
This flag is removed later.
Please be precise when providing an instruction or suggestion.

Reply:
Seems fine to me. The logically opposite meaning (“unavailable in Kubernetes v1.27 and later”) also makes sense if I imagine it.